### ISSUE TITLE
Feature request: Notepad++ like temp files

### ISSUE THREAD

**Issue by ggnkua** [slynch8/10x#780]
Let explain this feature:

- I am able to create a new file in notepad++
- I can freely edit it at will (with undo etc)
- I don't need to save it
- When I exit notepad++ the file is stored somewhere in its installation folders or temp folders
- When I restart notepad++ it restores the state of opened tabs, including unsaved files
- This also propagates to project files, so if I edit a .c file from a project and never save it, when I restart notepad++ it will remember that the file has been changed and also load the changes.

So in a nutshell notepad++ remembers changes in both saved and unsaved files across sessions. If this is tricky to do in 10x, I would at least appreciate if 10xnotepad could have this ability.

**Comment by slynch8** [comment]
done (0.35.60)
